src/Parquet.chpl

```diff
@@ -1,13 +1,28 @@
 module Parquet {
```

Comment:
Similar to the HDF5 module, I think changing this to ParquetMsg makes more sense now.